package/rpm-config-SUSE.spec

```diff
@@ -22,9 +22,9 @@ Version:        0
 Release:        0
 Summary:        SUSE specific RPM configuration files
 License:        GPL-2.0-or-later
-Group:          System/Packages
+Group:          Development/Tools/Building
```

Comment:
This is wrong, since this is *required* for package management to work, so the downstream group is wrong.